src/payment/strategies/googlepay/googlepay-cybersourcev2-initializer.ts

```diff
@@ -0,0 +1,113 @@
+import { round } from 'lodash';
+
+import { Checkout } from '../../../checkout';
+import { InvalidArgumentError } from '../../../common/error/errors';
+import PaymentMethod from '../../payment-method';
+
+import { BillingAddressFormat, GooglePaymentData, GooglePayInitializer, GooglePayPaymentDataRequestV2, TokenizePayload, TokenizeType } from './googlepay';
+
+export default class GooglePayCybersourceV2Initializer implements GooglePayInitializer {
+    initialize(
+        checkout: Checkout,
+        paymentMethod: PaymentMethod,
+        hasShippingAddress: boolean
+    ): Promise<GooglePayPaymentDataRequestV2> {
+        return Promise.resolve(this._getGooglePayPaymentDataRequest(
+            checkout,
+            paymentMethod,
+            hasShippingAddress
+        ));
+    }
+
+    teardown(): Promise<void> {
+        return Promise.resolve();
+    }
+
+    parseResponse(paymentData: GooglePaymentData): Promise<TokenizePayload> {
+        try {
+            const {
+            paymentMethodData: {
```

Comment:
indentation seems off 

also, `const` can be defined outside the `try` block so it's easier to read/follow